src/modules/TraceRouteModule.cpp

```diff
@@ -153,6 +153,20 @@ void TraceRouteModule::alterReceivedProtobuf(meshtastic_MeshPacket &p, meshtasti
     }
 }
 
+void TraceRouteModule::processUpgradedPacket(const meshtastic_MeshPacket &mp)
+{
+    if (mp.which_payload_variant != meshtastic_MeshPacket_decoded_tag || mp.decoded.portnum != meshtastic_PortNum_TRACEROUTE_APP)
+        return;
+
+    meshtastic_RouteDiscovery decoded = meshtastic_RouteDiscovery_init_zero;
+    if (!pb_decode_from_bytes(mp.decoded.payload.bytes, mp.decoded.payload.size, &meshtastic_RouteDiscovery_msg, &decoded))
+        return;
+
+    handleReceivedProtobuf(mp, &decoded);
+    meshtastic_MeshPacket copy = mp;
```

Comment:
The original packet will not be modified using this. It also can't, because it's a `const`.

Reply:
Oops, that was why I made it mutable before. I thought I was being smart by changing it.

Reply:
@h3lix1 This seems to be still there. `const` should be removed and then it should modify the original struct.